src/js/_enqueues/wp/updates.js

```diff
@@ -2674,6 +2674,17 @@
 			);
 		});
 
+		/**
+		 * Click handler for disabled links.
+		 *
+		 * @since 6.5.0
+		 *
+		 * @param {Event} event Event interface.
+		 */
+		$document.on( 'click', '.button-disabled', function( event ) {
+			event.preventDefault();
+		} );
```

Comment:
@afercia Hmm, removing the `href` entirely means it isn't focusable, or at least I can't tab to it.

If I use an empty `href`, that works, but without the `event.preventDefault()`, it's still clickable and refreshes the page.

What about this?

- Change the `href` to `''`.
- add `role="button'` even if it's weird that a link changes to a button dynamically, as said this is a temporary fix
- add `aria-disabled="true"`
- keep `event.preventDefault()`

Reply:
> link changes to a button dynamically

Could the Install Now and Activate Now buttons have the `aria-button-if-js` class when the page loads? Is there still a scenario in which one of them opens a separate page (other than purposely opening in a new tab)?

In 6.4, Update Now already had the `aria-button-if-js` class to switch the `role` to `button`. Clicking that (or pressing Enter) after updating a plugin does nothing, and the button does not have the `aria-disabled` or focus indication either.

Reply:
> @afercia Hmm, removing the `href` entirely means it isn't focusable, or at least I can't tab to it.

🤦🏽‍♂️  I forgot that would happen.

To recap:

Previously on WordPress 6.4:
When clicking 'Activate' after installing a new plugin on the `plugin-install.php` page, users were redirected to the `plugin.php` page. As such, there was no risk to have an 'Activate' link that wqs still clckable.

On trunk:
The [increased AJAX-ified experience added by the Plugin Dependencies feature](https://core.trac.wordpress.org/ticket/60663) (as referred to in the Trac ticket) introduces a new problem that I'd call a regression. After Installing a new plucin on the `plugin-install.php` page there's no redirect to the `plugin.php` page any longer. As such, the Activate link is still on the page and it's still clickable.
Ideally, there should have been a larger refactor of all these links. 
Historically, these are link to support the 'no-js' scennario. I wonder whether that makes still sense: should we still support a 'no-js' scenario?
Controls that behave like buttons should be buttons.
When JS is on, all these links now behave like buttons. They should have been refactored to buttons.
That said, at this point we can only implement a temporary fix, which is less than ideal.

> If I use an empty href, that works, but without the event.preventDefault(), it's still clickable

Well I can think of two options, both hacky:
- Remove the href entirely and add `tabindex="0" role="button" aria-disabled="true"`
- Keep the href add `role="button" aria-disabled="true"` and use `event.preventDefault()`

Reply:
@afercia It seems like "Remove the `href` entirely and add `tabindex="0" role="button" aria-disabled="true"` may be the least undesirable as a temporary fix as it doesn't rely on JavaScript.

I totally agree that a wider overall of the links vs buttons is needed in 6.6 and beyond.

On the "regression", I would agree that it's a regression from an accessibility perspective due to the markup of the link/button. Once that's improved, I think the removal of the automatic redirect is an enhancement that will benefit all users to be able to install and activate multiple plugins without leaving their current context. As plugins can be installed numerous ways, onboarding experiences shouldn't rely on just this redirect to trigger the onboarding. FWIW: I'd actually be quite interested in exploring a "Configure" / "Set up" action that, if provided, would appear after the plugin is activated. This would remove the need for "takeover" onboarding experiences on page reload/navigation, and instead provide a direct action in the plugin row or plugin card for the user to select intentionally.

Reply:
It seems like this just needs a decision. I think we're all agreed that none of the available options in the short term are ideal. I think we should just move forward with one of these options and start addressing the wider issue in 6.7. 

I lean towards the second option, because the entire behavior of this link without an `href` is already dependent on JS - the argument that removing it doesn't rely on JS doesn't really make sense to me.

Reply:
FYI, I have a [partially written ticket for raising the issue of changing link-buttons into real buttons](https://docs.google.com/document/d/1lRBIjbduvUkoqPbZmtNugXo5wS_zbksbYu_BJ_Shkws/edit?usp=sharing). Just haven't had the time to finish it. I've just dropped it into a Google doc, if anybody wants to make comments. I wanted to make sure that the arguments were cogent and clear before actually opening the ticket.